server/controllers/stock/inventory.js

```diff
@@ -167,6 +170,37 @@ function getInventoryItemsById(req, res, next) {
   .done();
 }
 
+/**
+* GET /inventory/metadata/receipt
+* Returns a pdf file for inventory metadata
+*
+* @function getInventoryItemReport
+*/
+function getInventoryItemReport(req, res, next) {
+  'use strict';
+
+  let request = {
+    query : req.query,
+    enterprise : req.session.enterprise,
+    project : req.session.project
+  };
+
+  core.getItemsMetadata()
+  .then((rows) => {
+    return listReceipt.build(rows, request);
+  })
+  .then((result) => {
+    let renderer = {
+      'pdf'  : '"Content-Type" : "application/pdf"',
+      'html' : '"Content-Type" : "application/html"',
+      'json' : '"Content-Type" : "application/json"'
+    };
+    let headers = req.query && req.query.renderer ? renderer[req.query.renderer] : renderer['pdf'];
```

Comment:
`req.query` is defined by express and will always be defined.  A cleaner way to write this code would be:
```js
// default to PDF headers if no renderer provided
let headerKey = req.query.renderer || 'pdf';
let headers = renderer[headerKey];
```
... or, if you want to be super compact, you could just write:
```js
let headers = renderer[req.query.renderer || 'pdf'];
```

This comment, of course, won't block the PR.